aspnetcore/fundamentals/environments.md

```diff
@@ -125,84 +127,93 @@ The production environment should be configured to maximize security, performanc
 
 ## Setting the environment
 
-It's often useful to set a specific environment for testing. If the environment isn't set, it will default to `Production` which disables most debugging features.
+It's often useful to set a specific environment for testing. If the environment isn't set, it defaults to `Production`, which disables most debugging features. The method for setting the environment depends on the operating system.
 
-The method for setting the environment depends on the operating system.
+### Azure App Service
 
-### Azure
+To set the environment in [Azure App Service](https://azure.microsoft.com/services/app-service/), perform the following steps:
 
-For Azure app service:
-
-* Select the **Application settings** blade.
-* Add the key and value in **App settings**.
+1. Select the app from the **App Services** blade.
+1. In the **SETTINGS** group, select the **Application settings** blade.
+1. In the **Application settings** area, select **Add new setting**.
+1. For **Enter a name**, provide `ASPNETCORE_ENVIRONMENT`. For **Enter a value**, provide the environment (for example, `Staging`).
+1. Select **Slot Setting** check box if you wish the environment setting to remain with the current slot when deployment slots are swapped. For more information, see [Azure Documentation: Which settings are swapped?](/azure/app-service/web-sites-staged-publishing).
```

Comment:
~~Select **Slot Setting** check box if you wish the environment setting to remain with the current slot when deployment slots are swapped.~~
Select the **Slot Setting** checkbox to preserve the environment setting in the current slot when the deployment slots are swapped.

Reply:
* 5.5.20: "Use check box, not box or checkbox, to refer to a check box in UI."
* :apple: to :apple: on the rest imo (e.g., "you" has been *un-banned* on the repo :smile:)
* Do need a def article there. :+1: